gateway-service/src/test/java/org/zowe/apiml/gateway/security/login/ProvidersTest.java

```diff
@@ -117,47 +121,63 @@ void givenZosmfIsRegisteredButUnavailable() {
     }
 
     @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Why is this needed?

Reply:
That line allows the method source to not be static, which it can't be static because it is part of a non-static nested class, and it can't be static and outside of the nested class as `@MethodSource` doesn't pick it up.